### PR TITLE
Playground fixes

### PR DESCRIPTION
**Pull request by javierFernandezMendez**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { NgModule } from '@angular/core';
+import { MAT_FORM_FIELD_DEFAULT_OPTIONS } from '@angular/material';
```
Comment (daniel-grana) [inline src/app/app-routing.module.ts:2]: This import is not necessary here isn't it? Remove it please

**Review by daniel-grana — CHANGES_REQUESTED** [review]

File: src/app/main/modals/dialogs/dialogs.component.ts [hunk 2]
```diff
@@ -116,7 +123,6 @@ export class DialogsComponent implements OnInit {
   showInfo(evt: any) {
     if (this.dialogService) {
       const config: ODialogConfig = {
-        icon: 'alarm',
         okButtonText: 'It rocks!'
```
Comment (daniel-grana) [inline src/app/main/modals/dialogs/dialogs.component.ts:126]: Do not override okButtonText on standard dialogs (review all examples)

**Review by daniel-grana — CHANGES_REQUESTED** [review]

**Review by patricia-martinez-imatia — DISMISSED** [review]
1. This screen break in 1270px resolution

image
2. on the table and list screens no, the configuration button is not aligned with the quickfilter, on the other hand on the grid screens yes

File: src/app/main/data/table/table.component.scss [hunk 3]
```diff
@@ -16,6 +16,7 @@ example-component {
 }
 
 table-basic {
+  height: calc(100% - 67px);
```
Comment (patricia-martinez-imatia) [inline src/app/main/data/table/table.component.scss:19]: Why this height? the table basic screen keeps showing up  unnecessary scroll  
image

**Review by patricia-martinez-imatia — CHANGES_REQUESTED** [review]

File: src/app/shared/example/example.component.scss [hunk 4]
```diff
@@ -46,10 +47,11 @@
 
   .playground-examples {
     margin: 20px 25px 10px 25px;
+    height: calc(100% - 350px);
```
Comment (patricia-martinez-imatia) [inline src/app/shared/example/example.component.scss:50]: Why this height? before it had a height of 350px

**Review by patricia-martinez-imatia — CHANGES_REQUESTED** [review]

File: src/app/shared/example/example.component.scss [hunk 5]
```diff
@@ -8,6 +8,7 @@
     padding: 0;
     margin: 0 10px 0 10px;
     width: calc(100% - 20px);
+    height: 50px;
```
Comment (patricia-martinez-imatia) [inline src/app/shared/example/example.component.scss:11]: why this height? I think this component height should be flexible but max-height:64px

**Review by patricia-martinez-imatia — CHANGES_REQUESTED** [review]
- when you switch from light to dark mode several times the result of the input home is as follows. I don't know what are the steps to reproduce it

image

File: src/app/main/data/table/table.component.scss (hunk 3 above)
Comment (javierFernandezMendez) [reply src/app/main/data/table/table.component.scss:19]: https://github.com/OntimizeWeb/ontimize-web-ngx-playground/issues/185

**Review by daniel-grana — DISMISSED** [review]

**Review by patricia-martinez-imatia — APPROVED** [review]

**Review by daniel-grana — APPROVED** [review]

**Pull request merged**